oauth2client/contrib/gce.py

```diff
@@ -16,60 +16,117 @@
 
 Utilities for making it easier to use OAuth 2.0 on Google Compute Engine.
 """
-
+import datetime
 import json
 import logging
 import warnings
 
 import httplib2
 from six.moves import http_client
-from six.moves import urllib
 
-from oauth2client._helpers import _from_bytes
-from oauth2client import util
 from oauth2client.client import HttpAccessTokenRefreshError
 from oauth2client.client import AssertionCredentials
-
+from oauth2client._helpers import _from_bytes
 
 __author__ = 'jcgregorio@google.com (Joe Gregorio)'
 
 logger = logging.getLogger(__name__)
 
 # URI Template for the endpoint that returns access_tokens.
-_METADATA_ROOT = ('http://metadata.google.internal/computeMetadata/v1/'
-                  'instance/service-accounts/default/')
-META = _METADATA_ROOT + 'token'
-_DEFAULT_EMAIL_METADATA = _METADATA_ROOT + 'email'
+_METADATA_ROOT = 'http://metadata.google.internal/computeMetadata/v1/'
 _SCOPES_WARNING = """\
-You have requested explicit scopes to be used with a GCE service account.
-Using this argument will have no effect on the actual scopes for tokens
-requested. These scopes are set at VM instance creation time and
-can't be overridden in the request.
+You have requested explicit scopes to be used with a GCE service account, and
+these scopes *are* present on the credentials. However, this request will have
```

Comment:
"However, your request"? This text reads awkwardly in the way it uses the word "request" to mean both "thing the user asked to happen" and "a software object representing a yet-to-be-transmitted HTTP request".

Also what is "the credentials"? Should the previous line read "a GCE service account credentials" rather than "a GCE service account"?